msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -9,13 +9,34 @@ ms.localizationpriority: medium
 
 # Build apps for anonymous users
 
-You can build bots, messaging extensions, cards and task modules in your app to engage with anonymous meeting participants.
+You can build tabs and task modules in your app to engage with anonymous meeting participants.
+
+To build apps for anonymous users ensure the following:
+
+* [Enable `supportsAnonymousGuestUsers` to true in your app manifest](#manifest-update-for-supportsanonymousguestusers).
+* [Admin setting for anonymous user app interaction](#admin-setting-for-anonymous-user-app-interaction).
+
+## Manifest update for `supportsAnonymousGuestUsers`
+
+To allow anonymous users to interact with the tab app, ensure to update your app manifest `meetingExtensionDefinition` property with `supportsAnonymousGuestUsers` to `true`. Following is an example of the manifest:
+
+```json
+
+ "meetingExtensionDefinition": {
+  "supportsAnonymousGuestUsers": true
+ }
+
+```
 
 To test your app's experience for anonymous users, select the URL in the meeting invite and join the meeting from a private browser window.
 
+## Anonymous user authentication flow
+
+As anonymous users aren't AAD accounts, skype token isn't available for anonymous users in third-party apps. This means that if `getAuthToken` is called in client SDK it would fail. Ensure that your app identifies the anonymous users and provides the correct experience in the meetings. You can use [Get user context](~/tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library) API to determine whether user is anonymous or not and redirect to the SSO page of the app.
```

Comment:
Developers will most likely never have heard of "skype token" in the context of AAD Accounts or Teams Authentication, can we remove this line?

Something like the following might be clearer:
> Anonymous users do not have an AAD account so using AAD authentication or `getAuthToken` from the client SDK will fail. If you need to authenticate anonymous users, your app should instead identify anonymous users and provide an alternative authentication experience for them in the meetings. You can determine if a user is anonymous by looking at the user's context (link to the docs on context below).

Reply:
Updated with the reworded content